media-sound/gtick/files/0.5.4-fno-common.patch

```diff
@@ -0,0 +1,13 @@
+Bug: https://bugs.gentoo.org/709396
+
```

Comment:
Is it a patch from upstream? If so, please use `git format-patch` for full patch header if possible, and add the Gentoo bugref as a comment in the ebuild instead.

Reply:
This is my patch. There is no upstream activity since 2014, neither in the git, nor in the mailing list.

Reply:
oh my, another one of those ;) well, at least that is 4 years later than gtklick.

please mark both patches as downstream patches in their respective headers, and mention why it is probably not worth trying to contact upstream.